.github/workflows/deploy.yml

```diff
@@ -18,49 +18,34 @@ jobs:
         tar -xf go1.13.14.linux-amd64.tar.gz
         sudo mv go /usr/local
         make release
+        cp irgsh-go/usr/bin/irgsh-cli target/irgsh-cli
 
-    - name: Deploy to rani
-      env:
-        HOST: ${{secrets.RANI_HOSTNAME}}
-        KEY: ${{secrets.RANI_DEPLOYMENT_KEY}}
-        USER: ${{secrets.RANI_DEPLOYMENT_USER}}
-        ARGS: -azvHP --exclude=/.git/
-        SSH_ARGS: '-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no'
-      run: |
-        eval $(ssh-agent -s)
-        ssh-add <(echo "$KEY")
-        rsync $ARGS -e "ssh $SSH_ARGS" target/release.tar.gz $USER@$HOST:/home/$USER/irgsh/release.tar.gz
-
-    - name: Update binary irgsh in rani
-      env:
-        HOST: ${{secrets.RANI_HOSTNAME}}
-        KEY: ${{secrets.RANI_DEPLOYMENT_KEY}}
-        USER: ${{secrets.RANI_DEPLOYMENT_USER}}
-        SSH_ARGS: '-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no'
-      run: |
-        eval $(ssh-agent -s)
-        ssh-add <(echo "$KEY")
-        ssh $SSH_ARGS $USER@$HOST -T << EOF
-        set -xe
-        sudo systemctl stop irgsh-chief
-        sudo systemctl stop irgsh-builder
-        sudo systemctl stop irgsh-repo
+    - uses: actions/upload-artifact@master
+      with:
+        name: irgsh-cli
+        path: target/irgsh-cli
 
-        pushd /home/deployment/irgsh/
-        tar -xvzf release.tar.gz
-        sudo cp -v irgsh-go/usr/bin/irgsh-* /usr/bin/
-        rm -rf release.tar.gz
-        rm -rf irgsh-go/
-        echo `date +%Y-%m-%d_%H:%M:%S` > latest-deployment.txt
-        popd
+    - uses: actions/upload-artifact@master
+      with:
+        name: release
+        path: target/release.tar.gz
 
-        sudo systemctl start irgsh-chief
-        sudo systemctl start irgsh-builder
-        sudo systemctl start irgsh-repo
-        EOF
+    - name: Create release
+      uses: Roang-zero1/github-create-release-action@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Create GitHub release
+      uses: Roang-zero1/github-upload-release-artifacts-action@master
+      with:
+        args:
+        - target/irgsh-cli
+        - target/release.tar.gz
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-    - name: Send notification to Telegram
+    - name: Deploy to rani
       env:
-        JIN_BOTOL_KEY: ${{secrets.JIN_BOTOL_KEY}}
+        HOST: ${{secrets.RANI_HOSTNAME}}
+        KEY: ${{secrets.JIN_IPRIT}}
       run: |
-        curl -X POST -H 'Content-Type: application/json' -d '{"chat_id": "-214965156", "text": "Perkakas IRGSH yang baru telah berhasil dideploy ke rani.boi.", "disable_notification": true}' https://api.telegram.org/bot$JIN_BOTOL_KEY/sendMessage
+        curl --header "Content-Type: application/json" --request POST --data '{"name":"irgsh","token": "$KEY","command":"./test-deploy.sh"}' https://$HOST
```

Comment:
`command` is not recognized. The value that can be passed is `value`, eg: branch name.  Please define the script in the yaml instead.

We should not passing direct command to avoid arbitrary command execution.

Reply:
okay pak @herpiko i'll fix that